gallery.html

```diff
@@ -45,7 +47,7 @@ <h2>Gallery</h2>
       <br /><br />
       <main>
         <div class="gallery">
-          <a row-span="2" href="https://loremflickr.com/1024/1024/?lock=50" >
+          <a class="row-span-2" href="https://loremflickr.com/1024/1024/?lock=50" >
```

Comment:
There are again quite a few merging issues here.  Don't be shy to ask for help in getting merges right, as it's important for having things in place for future labs